pinpoint-hbase/hbase-site-standalone.xml

```diff
@@ -0,0 +1,42 @@
+<configuration>
+    <property>
+        <name>hbase.rootdir</name>
+        <value>file:///home/pinpoint/hbase</value>
+    </property>
+    <property>
+        <name>hbase.cluster.distributed</name>
+        <value>true</value>
+    </property>
```

Comment:
if it's stand-alone mode. this should be removed

Reply:
It is necessary, otherwise region server ipc port will be randomized whenever the hbase container runs.

Reply:
why does `region server ipc port` not be randomized?

Reply:
The pinpoint collector should connect to hbase region server ipc. 
I found a similar problem on stackoverflow.

https://stackoverflow.com/questions/43524733/how-to-specify-rpc-server-port-for-hbase